src/Agent.Plugins/BuildDrop/BuildDropClient.cs

```diff
@@ -0,0 +1,74 @@
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.TeamFoundation.Build.WebApi;
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
+using Microsoft.VisualStudio.Services.BlobStore.Common;
+using Microsoft.VisualStudio.Services.Content.Common.Tracing;
+using Microsoft.VisualStudio.Services.BlobStore.WebApi;
+using Microsoft.VisualStudio.Services.Common;
+using Microsoft.VisualStudio.Services.WebApi;
+using Microsoft.VisualStudio.Services.Agent.Util;
+using Newtonsoft.Json;
+using Agent.Sdk;
+
+namespace Agent.Plugins.BuildDrop
+{    
+    // A wrapper of BuildDropManager, providing basic functionalities such as uploading and downloading drop artifacts.
+    public class BuildDropClient
```

Comment:
base on name convention, you might want to call this `BuildDropServer`

Reply:
renamed